src/applications/disability-benefits/2346/components/emptyField.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const emptyField = () => <div />;
+
+export default emptyField;
```

Comment:
I couldn't find where this was being used...what is it for??

Reply:
This has been removed